From 664e4258daba18facc782f388b70a8f398e06187 Mon Sep 17 00:00:00 2001
From: zhouxl <zhouxl@spacemit.com>
Date: Mon, 30 Dec 2024 10:43:42 +0800
Subject: [PATCH 1420/1448] efivar: add efivar filesystem support

After enable efivar filesystem support in kernel, we can access
efi variable through path /sys/firmware/efi/efivars.

efi variable include boot order, boot timeout, boot sequence...,
we can modify its value through efivars in kernel command line
without UEFI bootloader code.

Change-Id: If52dc5f06ae940fdaf407327966693c54f759dc9
---
 arch/riscv/configs/k1_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/configs/k1_defconfig b/arch/riscv/configs/k1_defconfig
index 3fff8b1c0a89..b1e2df8f73e7 100644
--- a/arch/riscv/configs/k1_defconfig
+++ b/arch/riscv/configs/k1_defconfig
@@ -1121,7 +1121,7 @@ CONFIG_PROC_KCORE=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
 CONFIG_HUGETLBFS=y
-# CONFIG_EFIVAR_FS is not set
+CONFIG_EFIVAR_FS=y
 CONFIG_ORANGEFS_FS=m
 CONFIG_ADFS_FS=m
 CONFIG_AFFS_FS=m
-- 
2.47.0

